### DISCUSSION TITLE
歐盟規定充電器規格一律改爲type-c

### DISCUSSION THREAD

**Discussion by ABCDEFGHIJKLMNO12345678** [1101-GS4719-C/Python-Community-GS4719-C#738] [General]
現今社會 充電器的種類相當多元 有蘋果用的lightning type-C USB-C... 有時爲了充電不斷更換接頭相當麻煩 歐盟也投票在未來一律改爲type-c 不僅方便了世人 對環境也更佳的友善

https://tw.news.yahoo.com/%E6%AD%90%E6%B4%B2%E8%AD%B0%E6%9C%83%E9%80%9A%E9%81%8E%E6%96%B0%E6%B3%95-%E8%A6%8F%E5%AE%9A%E5%85%85%E9%9B%BB%E5%99%A8%E8%A6%8F%E6%A0%BC%E7%B5%B1-%E4%BD%BF%E7%94%A8type-c-001152194.html

**Comment by Zoe06060606** [comment]
統一接頭真的比較方便
不然包包都要放很多條充電線超麻煩🥲

**Comment by danwan1027** [comment]
type c 終究要統一天下了嗎！

**Comment by dinosaur1009** [comment]
感覺是個好政策 但會經歷汰換的陣痛期....

**Comment by Yunhans** [comment]
希望iphone也能用type c

**Comment by AnAnistired** [comment]
終於啊，科技產品規格化不但消費者使用方便，也能減少更多科技廢棄物的產生，省錢又方便。（lighting真的該下去了

**Comment by moonnight3503** [comment]
大多數電子產品都是type-c，如果可以統一的話，感覺真的蠻方便的，不同廠牌產品也可以通用。

**Comment by manipulateed** [comment]
終於要整合，執行一個標準化的生產規定了，這讓生活更加方便了呢。

**Comment by ghost** [comment]
整合雖然會變得更加方便，但type C真的有比Lightning好嗎？我認為如果Lightning性能或發展性上比typeC好，那不就是在捨本逐末。所以與其統一，不如讓市場自由選擇，這樣對產業發展才是比較好的吧。至於環保問題，致力於改善產品的可回收，與消費者的環保意識，才是從根本解決問題吧

**Reply by AnAnistired** [reply]
lightning 已經極限了，快充速度跟不上安卓機

**Comment by HuangHsinTzu** [comment]
我很喜歡這個想法
統一充電頭可以避免需針對不同產品製造相對應的充電頭
造成資源浪費
也讓我們在旅遊時更方便，不用帶一大堆線

**Comment by Jasper525** [comment]
統一用type c是好事，希望充電協議也能慢慢統一，才不用一次帶一條type c線加好幾顆充電頭，或必須買支援多種協議的副廠GaN快充頭。

**Comment by ruthchen1020** [comment]
TypeC有什麼強勢之處嗎？對各種連接線的優劣勢不太了解。

**Comment by evanchou1204** [comment]
對消費者來說會很方便沒錯，但對蘋果來說，這樣限制他們的轉接頭確實不太符合比例原則，也讓他們很難做生意。

**Comment by Pietra1226** [comment]
我也支持轉type-C，可以節約能源而且讓我們只要帶著一條線就好，節省了很多時間成本

**Comment by a45320614** [comment]
現在ipad系列產品都已經是type-C了，不知道為何iphone堅持用lightning，都用type-C的話會對使用者比較方便，也比較環保!

**Comment by williamyen0623** [comment]
拜託iphone趕快換type-C，桌上一堆線快受不了了

**Comment by dabbie109303501** [comment]
歐洲Iphone的使用者要哭哭了

**Comment by yuxinlin93** [comment]
從去年買的ipad就是type-C接頭，如果沒帶到蘋果的充電器，用別廠的充電器也可以充，是蠻方便的，但舊款蘋果用戶真的是會麻煩到ㄋㄟ

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
早該如此了 加上type-c充電效率又更高 不盡如此type-c又能更適用於各個場牌

**Comment by CPCpyt** [comment]
統一接頭對於產品的標準化有很大的助力，
不過對於致力發展產品差異化的廠商，例如APPLE可能就會面臨比較大的衝擊。

**Comment by swerfg03joe** [comment]
工業本來就是會有規格化的出現，會想堅持自家產品的，總是拿統一規格會阻礙創新發展當作藉口，但多半是因為商業考量，線即是一種耗材。當一家公司要賣電子產品，卻連充電線的錢也要壟斷。只是因為APPLE是美國的公司所以才可以賣那麼久還不被針對

**Comment by ru0372572** [comment]
lightning真的超不方便，每次出門都要帶兩條充電線

**Comment by Guagua1234** [comment]
同意統一變成type-c，
lightning的充電速度不如type-c，
且iphone和ipad的充電線不一樣，
這不就代表lightning已經無法再提高了嗎，
就趁這個機會一起統合在一起。
也比之後蘋果廠商自己改回來type-c來的得體，
也減少被罵的可能性。

**Comment by Valentinaa0214** [comment]
全部改成type-c超讚的，
出門就不用帶一大堆亂七八糟的線或轉接頭，
一條線壞掉也不用急著買新的，可以用其他產品的充電頭撐一下之類的，
希望iPhone15是type-c阿阿阿阿。

**Comment by s96015welson** [comment]
我認為統一成type-c十分方便，可以減少因產品更換產生的電子廢棄物，不過有個疑慮是現在的type-c太強大了，甚至可以用在高功率的充電器上，不過假如此時使用的傳輸線品質不夠好(例如拿副廠的手機的傳輸線直接使用在筆電上)，可能會有安全上的隱憂，然而充電孔都統一的同時，就更有可能發生此種情況。

**Comment by Twenty0522** [comment]
每次出門玩要帶許多不同的線材也覺得滿麻煩的，不過蘋果在很早就採用lightning雙頭皆可使用的線材是值得讚許的發明，從早期帶給用戶許多便利。然而lightning是蘋果的專利，而其他電子產品後續只能使用type-c，而我想蘋果在筆電已經改用type-c但手機卻尚未跟進有其設計考量，我認為應該給蘋果一點轉換時間或通融。

**Comment by Sylvie0601** [comment]
完全同意統一變成type-c，我是蘋果平板加安卓手機剛好都是type-c，以前是蘋果手機都要換兩條線充電超級麻煩，希望蘋果手機也可以變成type-c就好了!之前每次都要帶兩條線出門真的超級麻煩的。

**Comment by zephyrxin01** [comment]
我覺得統一的確比較好，至少出門在外不用帶一堆線，如果有人沒有帶線也能夠相互支援一下。

**Comment by alstinlin** [comment]
支持統一換成typec，不僅可以減少換充電線的開銷，而且我本身比較常忘了帶東西，如果統一換成typec就可以在家裡或書包都放一條備用的線，這樣一來還蠻方便的

**Comment by pudding0803** [comment]
每次都要準備很多種線🥲，因為線兩邊的頭不一定都能符合，尤其是剛好忘記帶充電線的話，還要看誰的充電線剛好跟自己一樣才能借，滿麻煩的😅，不過滿好奇這個政策是否真的會成功，感覺 Apple 都會特立獨行🤔

**Comment by Tzuhsuannnnn** [comment]
根據chat gpt給我的資訊，type-c 像USB 3.1 Gen 1的最大傳輸速度為5 Gbps（即625 MB/s），而USB 3.1 Gen 2的最大傳輸速度則為10 Gbps（即1.25 GB/s）。而Thunderbolt 3則提供了更高的傳輸速度，可達到40 Gbps（即5 GB/s）。但lightening卻只有480 Mbps（即60 MB/s）。因此我認為換type-c是好的，不論是速度上的提升，或是統一規格減少浪費，都是不錯的。

**Comment by sumairu0727** [comment]
雖然蘋果最後會改成type-c的連接方式，不過我認為蘋果還是會跟三星一樣發展特殊的快充協議(25W快充)，原廠充電設備應該還是會有需求的。
至於某樓說強制改type-c不符合比例原則這點我是不認同，畢竟工業本來就會有統一規格的趨勢，再來歐盟是基於環保的概念出發，如果說基於環保概念限制不符合比例原則，那說實話也不用去限制工業汙染了?

**Comment by ashleyy-huang** [comment]
Type-c實在是太方便了，可以同時傳輸影片、音軌，甚至是可以同時充電，且比現在蘋果的ligtening快非常多，很希望蘋果可以趕快把手機和airpods改成type-c不然每次出門都還需要戴上不一樣的充電器，非常的不方便，充電速度又比較慢

**Comment by bestbanana295** [comment]
我認為這是一個好政策，因為如果各個廠商都推出自己設計的充電器，那麼要換家用的時候就會造成浪費，如果都改成Type-c的話就方便多了。
